sigs.yaml

```
@@ -1912,6 +1912,54 @@ sigs:
   - name: scheduler-plugins
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/scheduler-plugins/master/OWNERS
+- dir: sig-security
+  name: security
```

```suggestion
  name: Security
```

one small nit, this ensures proper formatting of auto-generated markdown

---

Thank you.   I'm struggling with the rebase, because the messages I'm getting are about auto-generated files.  I'll figure this out...